src/Admin/Controllers/HomeController.cs

```diff
@@ -42,21 +44,28 @@ public IActionResult Error()
             });
         }
 
-        public async Task<IActionResult> GetLatestDockerHubVersion(string repository, CancellationToken cancellationToken)
+        public async Task<IActionResult> GetLatestVersion(string repository, CancellationToken cancellationToken)
         {
-            var requestUri = $"https://hub.docker.com/v2/repositories/bitwarden/{repository}/tags/";
+            var requestUri = $"https://raw.githubusercontent.com/bitwarden/self-host/master/version.json";
             try
             {
                 var response = await _httpClient.GetAsync(requestUri, cancellationToken);
                 if (response.IsSuccessStatusCode)
                 {
                     using var jsonDocument = await JsonDocument.ParseAsync(await response.Content.ReadAsStreamAsync(cancellationToken), cancellationToken: cancellationToken);
                     var root = jsonDocument.RootElement;
-
-                    var results = root.GetProperty("results");
-                    foreach (var result in results.EnumerateArray())
+                    var versionsNode = root.GetProperty("versions");
+                    if (repository == "web")
+                    {
+                        var name = versionsNode.GetProperty("webVersion").GetString();
+                        if (!string.IsNullOrWhiteSpace(name) && name.Length > 0 && char.IsNumber(name[0]))
```

Comment:
I think this validation was to avoid the special named version from docker like `latest`, `rc`, and `dev`. So since we are switching to our own source of truth we can probably skip doing this validation. We can also probably have a static `Dictionary<string, string>` to map from `api -> coreVersion` and `web -> webVersion`.